src/command.ts

```diff
@@ -199,14 +199,24 @@ export default abstract class Command {
     return Errors.error(input, options as any)
   }
 
-  log(message = '', ...args: any[]): void {
+  stdout(message = '', ...args: any[]): void {
```

Comment:
I think we should just keep this as `log`. I don't want to confuse anyone by using the name `stdout` since that's already a well known method in a node process and this method doesn't behave exactly like it.